### PR TITLE
Added Form Child Component

### PR DESCRIPTION
**Pull request by GermainBergeron**
Fixed reload don’t go to the selected section

### CODE REVIEW

File: scss/forms/form-child.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
+$child-padding-y: 10px;
+$child-padding-x: 25px;
+
+.coveo-child {
+  padding: 0 $child-padding-x $child-padding-y;
+
+  background-color: $light-grey;
+  border-radius: 3px;
```
Comment (JPLachance) [inline scss/forms/form-child.scss]: Are you sure? It's 2px for modals and for other UI components.

File: scss/forms/form-child.scss [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    display: block;
+    width: calc(100% + ($child-padding-x * 2));
+    height: $child-padding-y;
+    margin: 3px (-1 * $child-padding-x) ($child-padding-y / 2);
+
+    content: '';
+
+    background-color: $light-grey;
+    border-top: ($child-padding-y / 4) * 3 solid $pure-white;
+    border-left: ($child-padding-y / 4) * 3 solid $light-grey;
```
Comment (JPLachance) [inline scss/forms/form-child.scss]: 👍

File: resources/js/scripts.js [hunk 3]
```diff
@@ -10,18 +15,16 @@ $(document).ready(function() {
...
     var selectedTab;
-    window.onload = function() {
-        if (window.location.hash) {
-            var selectedTabChild = $('.navigation-menu-section-item-link[href="' + window.location.pathname + window.location.hash + '"]');
-            selectedTab = selectedTabChild.parent().addClass('state-active');
-        } else {
-            var selectedTabParend = $('.navigation-menu-section-item-link[href*="' + window.location.pathname + '"]');
-            selectedTab = $(selectedTabParend[0]).parent().addClass('state-active');
-        }
+    if (window.location.hash) {
```
Comment (JPLachance) [inline resources/js/scripts.js]: Wow, thanks for that!

File: scss/forms/form-child.scss (hunk 1 above)
Comment (pochretien) [inline scss/forms/form-child.scss]: +1 for 2px

**Pull request merged**